cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -474,6 +475,7 @@ protected void configureRoutes() {
         get(forecastFilePath, new ForecastFileController(metrics));
         addCacheControl(forecastFilePath, 1, TimeUnit.DAYS);
 
+        get("/projects/children/", new ProjectChildrenHandler(metrics));
```

Comment:
child-locations?

Reply:
I can see child is not exactly right.  Do you have a better suggestion? They aren't sub-locations. Are they just locations and b/c the url is /projects/locations it will be clear that they are locations at the projects?

Reply:
Or are you suggesting the url be child-locations instead of children?

Reply:
I renamed to /projects/child-locations.

Reply:
just `locations` would probably be better but then it would make sense to be `projects/<project>/locations` which I don't think we're ready for yet.

Reply:
Correct Ryan, I was asking that location be added to the endpoint. I also like Mikes suggestion.

Reply:
I don't think I'm ready for Mike's suggestion.  I'll have to look but I don't think we've done that anywhere else yet. If we haven't done that anywhere else I guess I'm going to use /projects/locations